packages/atomic/src/utils/combobox.tsx

```diff
@@ -161,9 +161,10 @@ export class Combobox {
 
     this.removeEmptyOptionElement();
 
-    Array.from(this.listboxOptions).forEach((value) =>
-      this.updateOption(value)
-    );
+    Array.from(this.listboxOptions).forEach((value, index) => {
+      value.setAttribute('id', `${this.options.id}-suggestion-${index}`);
```

Comment:
It's a small preference, but I would prefer if this was moved inside `updateOption` and an `index`/`id` parameter was added to `updateOption`.